requirements.txt

```diff
@@ -0,0 +1,52 @@
+attrs==19.3.0
```

Comment:
Right now you have both a requirements.txt and a conda requirements doc -- we should pick *ONE* supported approach; having both means you have to update both every time, and it menas you're now forced to support TWO styles of development environment (rather than just telling developers they have to suck it up and use the one you pick)